ci-operator/step-registry/ipi/conf/osimage/ipi-conf-osimage-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -z ${PLATFORM} ]]; then
+    echo "The environment variable PLATFORM is not defined!"
+    exit 1
+fi
+
+if [[ -z ${CLUSTER_OS_IMAGE} ]]; then
+    if [[ ! -f "${SHARE_DIR}/cluster_os_image" ]]; then
+        echo "Unable to get CLUSTER_OS_IMAGE!"
+        exit 1
+    fi
+    CLUSTER_OS_IMAGE=$(< "${SHARE_DIR}/cluster_os_image")
```

Comment:
Is there any other place to write the `cluster_os_image` file?

Reply:
I would add this for future usage, but seems no need, removed.